addon/components/frost-radio-button-input.js

```diff
@@ -1,29 +1,30 @@
 import Ember from 'ember'
 import PropTypeMixin, {PropTypes} from 'ember-prop-types'
-import FrostEvents from '../mixins/frost-events'
 
 const {
   Component,
-  computed
+  computed: {
+    alias
+  }
 } = Ember
 
-export default Component.extend(FrostEvents, PropTypeMixin, {
+export default Component.extend(PropTypeMixin, {
```

Comment:
I believe this is an internal construct (radio buttons need to be part of a group), so my assumption is that we would have the events/proxy on the group interface